### PR TITLE
46 victory screen

### PR DESCRIPTION
**Pull request by chanson02**

### CODE REVIEW

File: dunwoody_disaster/views/victoryScreen.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        lbl = QLabel('Manage your inventory: ')
+        layout.addWidget(lbl)
+
+        loot_screen = CollectLootScreen(fight_controller.player, fight_controller.enemy.get_items())
+        loot_screen.set_callback(self.loot_selected_event)
+        layout.addWidget(loot_screen)
+
+    def loot_selected_event(self):
+        print('Player selected the loot they wanted')
+        # TODO: Return to map
```
Comment (chanson02) [inline dunwoody_disaster/views/victoryScreen.py]: Do we have an API to do this? @EngineeringDude123

Comment (chanson02) [reply dunwoody_disaster/views/victoryScreen.py]: I pulled your branch in order to get it working, so now this PR is also blocked by #32.

File: dunwoody_disaster/views/fightScreen.py [hunk 2]
```diff
@@ -155,46 +164,27 @@ def AddToQueue(self, action):
...
+            self.player2.set_magic(self.player2.curMagic)
+            self.player2.set_stamina(self.player2.curStamina)
+
+            self.fightFlag = False
+            self.fight_Btn.setEnabled(True)
+
+    def onWin(self):
+        vc = VictoryScreen(self.fightSequence)
+        self.stacked_layout.addWidget(vc)
+        self.stacked_layout.setCurrentWidget(vc)
```
Comment (chanson02) [inline dunwoody_disaster/views/fightScreen.py:190]: @MLHumphries I think you're in charge of implementing the fight button? If you are, you can call this function is the player defeats the enemy.

**Review by MLHumphries — APPROVED** [review]

File: dunwoody_disaster/views/fightScreen.py (hunk 2 above)
Comment (chanson02) [reply dunwoody_disaster/views/fightScreen.py:190]: Nevermind, I think I found where this goes.

**Review by NoahBartell — APPROVED** [review]
I suppose this looks alright

**Pull request merged**